lib/configuration_options.dart

```diff
@@ -60,9 +60,9 @@ const String enableAnnotationEditing = "enableAnnotationEditing";
 
 const String enableTextSelection = "enableTextSelection";
 
-const String enableBookmarkList = "enableBookmarkList";
+const String androidEnableBookmarkList = "androidEnableBookmarkList";
```

Comment:
Shouldn't this be `const String androidEnableBookmarkList = "enableBookmarkList";` and `const String androidEnableDocumentEditor = "enableDocumentEditor";`?

Reply:
It's just the string used as a key in configuration map. It matches `ANDROID_ENABLE_BOOKMARK_LIST` in `ConfigurationAdapter`.

Reply:
Ah ok! Thanks! I thought it should be the same as https://github.com/PSPDFKit/pspdfkit-flutter/pull/21/files#diff-1bce22d614b794c1bd9d086607a85820R67.